src/Surfnet/ServiceProviderDashboard/Application/Service/LoadEntityService.php

```diff
@@ -132,7 +132,7 @@ public function load($dashboardId, $manageId, Service $service, $sourceEnvironme
         // Convert manage entity to domain entity
         $domainEntity = Entity::fromManageResponse(
             $manageEntity,
-            $environment,
+            $sourceEnvironment,
```

Comment:
Can this somehow be verified in the `LoadEntityServiceTest`? Or alternatively, in a functional test?

Reply:
Done